charts/hcp-terraform-operator/values.yaml

```diff
@@ -120,3 +120,7 @@ serviceAccount:
   # -- The name of the service account to use.
   # If not set and create is true, a name is generated using the fullname template.
   name: ""
+
+# -- Specifies whether a Role-Based Access Control (RBAC) resources should be created (default set to true)
```

Comment:
Thank you for adding a doc comment!

Please make the following changes:

- Move this comment to the `create` line (directly above the `create` attribute).
- Remove `(default set to true)`. This part will be generated by the `helm-docs` make target.
- Run `make helm-docs` to update the Helm README file with the new attribute.

Thank you!

Reply:
done